docs/src/pages/en/media-airplay-button.md

```diff
@@ -2,12 +2,30 @@
 title: <media-airplay-button>
 description: Media Airplay Button
 layout: ../../layouts/MainLayout.astro
+source: https://github.com/muxinc/media-chrome/tree/main/src/js/media-airplay-button.js
```

Comment:
did we want to add this to the layout to automatically render this link on the pages?

Reply:
yes I thought to keep this for a next iteration. it's not that important